sra-tools/1.0/sra_dump.smk

```diff
@@ -0,0 +1,149 @@
+import pandas as pd
+import os
+
+configfile: "test_sra_dump.yaml"
```

Comment:
Should this be renamed to the `sra_dump.yaml` (without `test`)?

Reply:
Good catch - fixed.